src/Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
Cargo automatically switched from 3 to 4 in 1.84.0, but it's supposedly backwards compatible to 1.78.0.